addOns/pscanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -81,3 +81,8 @@ pscanalpha.sri-integrity.name=Sub Resource Integrity Attribute Missing
 pscanalpha.sri-integrity.desc=The integrity attribute is missing on a script or link tag served by an external server. The integrity tag prevents an attacker who have gained access to this server from injecting a malicious content. 
 pscanalpha.sri-integrity.soln=Provide a valid integrity attribute to the tag.
 pscanalpha.sri-integrity.refs=https://developer.mozilla.org/en/docs/Web/Security/Subresource_Integrity
+
+pscanalpha.jsfunction.name=Dangerous JS Functions
+pscanalpha.jsfunction.desc=A dangerous JS function seems to be in use that would leave the site vulnerable.
+pscanalpha.jsfunction.soln=
```

Comment:
```suggestion
pscanalpha.jsfunction.soln=See the references for security advice on the use of these functions.
```

Reply:
Or something more specific if you can think of it..... "Consider whether this code handles input in a manner which might lead to XSS, XSSI, Open Redirect, etc or rewrite it in a manner which does not require the use of unsafe functions"
<shrug>